shell/components/nav/TopLevelMenu.vue

```diff
@@ -82,15 +81,6 @@ export default {
       return this.$store.getters['prefs/get'](PINNED_CLUSTERS);
     },
 
-    sideMenuStyle() {
-      const globalBannerSettings = getGlobalBannerFontSizes(this.$store);
-
-      return {
-        marginBottom: globalBannerSettings?.footerFont,
-        marginTop:    globalBannerSettings?.headerFont
-      };
-    },
```

Comment:
This wasn't functional for multi-line banners and hasn't been. The height is arbitrary to the number of lines in the entered.

The layout would slightly break where part of the banner would be overlapped by the TopLevelMenu and the rest of the layout would get pushed down further than the TopLevelMenu.